modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/resources/META-INF/resources/js/components/ResultRankingsForm.es.js

```diff
@@ -308,32 +308,20 @@ class ResultRankingsForm extends Component {
 							.initialSXPBlueprintExternalReferenceCode,
 				  };
 
-			fetch(
+			fetchResponse(
 				`${
 					window.location.origin
 				}${Liferay.ThemeDisplay.getPathContext()}${
 					scopeInfo.fetchItemByIdUrl
 				}${scopeInfo.value}`,
-				{
-					credentials: 'include',
-					headers: new Headers({
-						'Accept-Language': Liferay.ThemeDisplay.getBCP47LanguageId(),
-						'x-csrf-token': Liferay.authToken,
-					}),
-					method: 'GET',
-				}
+				{}
```

Comment:
We should change this in a separate ticket, but I wanted to note that instead of passing an empty object, it'd be better practice to handle not passing any additional parameter in the `fetchResponse` function.

Reply:
@thektan Good idea, we can address this in a later ticket! 👍